src/test/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployerIntegrationTests.java

```diff
@@ -1213,4 +1786,82 @@ public void describeTo(Description description) {
 			}
 		};
 	}
+
+	private String getPodEnvironment(String deploymentId) throws InterruptedException {
```

Comment:
This is nice! :-)

Reply:
only way to verify the actual env was setup properly ;)